pkg/asset/cluster/aws/aws.go

```diff
@@ -95,3 +99,7 @@ func tagSharedVPCResources(ctx context.Context, clusterID string, installConfig
 func sharedTag(clusterID string) (string, string) {
 	return fmt.Sprintf("kubernetes.io/cluster/%s", clusterID), "shared"
 }
+
```

Comment:
nit: Should we move these (sharedTag, elbTag) above in var/const section. At least above their usage in the function `tagSharedVPCResources`

Reply:
We could. I was just using the existing style in the code.

Reply:
Simple nit picking. I really don't think it matters but I have seen in most of our code that the functions are C-Style where the caller sits below